### ISSUE TITLE
Cannot calculate mAP for validation data

### ISSUE THREAD

**Issue by Batserine** [pjreddie/darknet#1913]
I have both validation and training data set arranged properly. I am getting 0.01% mAP for validation set, whereas if I use train dataset for validation I am getting mAP of 99.17%.
You can see them below
![yolo_valid](https://user-images.githubusercontent.com/26717031/68380612-c32b6a00-0182-11ea-9cbc-12edc8b2d023.png)
![yolo_valid1](https://user-images.githubusercontent.com/26717031/68380631-ca527800-0182-11ea-9221-77fe5c10678b.png)
